design/codebases.go

```diff
@@ -179,6 +190,37 @@ var _ = a.Resource("codebase", func() {
 		a.Response(d.NotFound, JSONAPIErrors)
 		a.Response(d.Unauthorized, JSONAPIErrors)
 	})
+	a.Action("cheState", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.GET("/che/state"),
+		)
+		a.Description("Get che server state.")
+		a.Response(d.OK, func() {
+			a.Media(cheServerState)
+		})
+		a.Response(d.BadRequest, JSONAPIErrors)
+		a.Response(d.InternalServerError, JSONAPIErrors)
+		a.Response(d.NotFound, JSONAPIErrors)
```

Comment:
no, che-starter API does not handle not found namespace case - I will remove this error